tests/src/run.sh

```diff
@@ -28,6 +28,6 @@ mkdir -p ./logs
 
 export LD_LIBRARY_PATH=/usr/lib:/lib:/home/root:${MY_DIR}
 
-./ut-test -l ./logs -p ./assets/test_kvp.yaml $@
+./ut-test -a -l ./logs -p ./assets/test_kvp.yaml --gtest_output=xml:./logs/ut-test-report.xml $@
```

Comment:
how we run the test because `ut-main` is common should be identical between c/c++

so there shouldn't be gtest specific switches that what `-l` is for, xml should be output because we're running -a automated mode, `-b` basic mode is without xml output.

`-gtest_output=xml:./logs/ut-test-report.xml`